src/category_theory/monad/basic.lean

```diff
@@ -56,4 +56,51 @@ attribute [simp] comonad.left_counit comonad.right_counit
 notation `ε_` := comonad.ε
 notation `δ_` := comonad.δ
 
+/-- A morphisms of monads is a natural transformation compatible with η and μ. -/
+structure monad_hom (M N : C ⥤ C) [monad M] [monad N] extends nat_trans M N :=
```

Comment:
Would you mind dualising the things you've added here?

Reply:
Done! (Also added bundled comonads  in category_theory.monad.bundled, and the corresponding category instance.)